eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -182,3 +182,88 @@ terraform -chdir=./terraform destroy -auto-approve
+```
+$ ### Send messages through imperative.  
+$ curl -X POST http://localhost:8080/messages/imperative/staticalDestination?message=hello
+$ curl -X POST http://localhost:8080/messages/imperative/dynamicDestination?message=hello
+
+$ ### Send messages through reactive.
+$ curl -X POST http://localhost:8080/messages/reactive?message=hello
+```
+or when the app runs on App Service or VM
+```
+$ ### Send messages through imperative.
+$ curl -d -X POST https://[your-app-URL]/messages/imperative/staticalDestination?message=hello
+$ curl -d -X POST https://[your-app-URL]/messages/imperative/dynamicDestination?message=hello
+
+$ ### Send messages through reactive.
+$ curl -d -X POST https://[your-app-URL]/messages/reactive?message=hello
+```
+Verify in your app’s logs that a similar message was posted:
+```
+New message received: 'hello', partition key: 2002572479, sequence number: 4, offset: 768, enqueued time: 2021-06-03T01:47:36.859Z
+Message 'hello' successfully checkpointed
+```
+
+### [Using Batch Consuming](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#batch-consumer-support)
+
+To work with the batch-consumer mode, the property of spring.cloud.stream.bindings.<binding-name>.consumer.batch-mode should be set as true. When enabled, an org.springframework.messaging.Message of which the payload is a list of batched events will be received and passed to the consumer function.
+
+In this sample, users can try the batch-consuming mode by enable the "batch" profile and fill the "application-batch.yml". For more details about how to work in batch-consuming mode, please refer to the [reference doc](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#batch-consumer-support-2).
+
+### Set Event Hubs message headers
+
+Users can get all the supported EventHubs message headers [here](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#scs-eh-headers) to configure.
+
+### Resource Provision
+
+Event Hubs binder supports provisioning of event hub and consumer group, users could use [properties](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#resource-provision) to enable provisioning.
+
+### Partitioning Support
+
+A PartitionSupplier with user-provided partition information will be created to configure the partition information about the message to be sent, [Here](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#partitioning-support-2) is the process of obtaining different priorities of the partition ID and key.
+
+#### Partition key support
+The binder supports Event Hubs partitioning by allowing setting partition key and session id in the message header. This section introduces how to set partition key for messages.
```

Comment:
Sorry there is wrong information I pass to you. There are some differences for partitioning support between Event Hubs and Service Bus. Event Hubs only support partition key and id, so there is no session id for eh. And the way to configure partition information is also different from sb, so let's change these several parts:
1. Change the title from "Partition key support" to "Partition support".
2. Delete below content and code, only remain "The binder supports Event Hubs partitioning by allowing setting partition key and id. Please refer to the [reference doc](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#partitioning-support-2) for more details."
3. Remove below section of "Session support"